exercises/concept/inventory-management/.docs/hints.md

```diff
@@ -4,32 +4,32 @@
 
 - [The Python Dictionary Tutorial](https://docs.python.org/3/tutorial/datastructures.html#dictionaries) can be a great introduction.
 
-## 1. Create the dictionary
+## 1. Create an inventory based on a list
 
 - You need a [for loop](https://docs.python.org/3/tutorial/controlflow.html#for-statements) to iterate the list of items, then insert each item in the dictionary if missing and increment the item count using the dictionary accessor.
 - You can use [`setdefault`](https://www.w3schools.com/python/ref_dictionary_setdefault.asp) to make sure the value is set before incrementing the count of the item.
 - This function should [return a dict](https://www.w3schools.com/python/ref_keyword_return.asp).
 
-## 2. Add items to a dictionary
+## 2. Add items from a list to an existing dictionary
 
 - You need a [for loop](https://docs.python.org/3/tutorial/controlflow.html#for-statements) to iterate the list of items, then insert each item if not already in the dictionary and [increment](https://www.w3schools.com/python/gloss_python_assignment_operators.asp) the item count using the dictionary accessor.
 - You can use [`setdefault`](https://www.w3schools.com/python/ref_dictionary_setdefault.asp) to make sure the value is set before incrementing the count of the item.
 - The function `add_items` can be used by the `create_inventory` function with an empty dictionary in parameter.
 - This function should [return a dict](https://www.w3schools.com/python/ref_keyword_return.asp).
 
-## 3. Decrement items in the dictionary
+## 3. Decrement items from the inventory
```

Comment:
The item must be in the inventory in order to decrement it, so that is perhaps as clear.